### DISCUSSION TITLE
BookmarkAdd type is missing in carbon icons

### DISCUSSION THREAD

**Discussion by gazpachu** [DefinitelyTyped/DefinitelyTyped#57008] [Issues with a @types package]
The BookmarkAdd icon type is missing in carbon icons: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/carbon__icons-react

[a comment by an automated account is not shown]

**Comment by metonym** [comment]
The types have been updated to [v11.5.0](https://www.npmjs.com/package/@types/carbon__icons-react).

**Comment by metonym** [comment]
As of version 11.29.0, `@carbon/icons-react` ships with its own TypeScript definitions.

As such, `@types/carbon__icons-react` is no longer needed and has [been removed](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67176) as of v11.29.0.
